### DISCUSSION TITLE
New links

### DISCUSSION THREAD

**Discussion by neverfa11ing** [djsime1/awesome-flipperzero#8] [Show and tell]
I've got 2 new repos that would fit here if wanted.

https://github.com/neverfa11ing/FlipperAmiibosNFC
https://github.com/neverfa11ing/FlipperMusicRTTTL

**Comment by djsime1** [comment]
RTTTL was already added, and I don't see the need to add another Amiibo dump.

**Comment by V0lk3n** [comment]
https://github.com/V0lk3n/Flipper-Skylanders

If you want to add it :)

**Reply by djsime1** [reply]
Sure.

**Reply by V0lk3n** [reply]
> Sure.

Thanks! :)

**Reply by V0lk3n** [reply]
I've submitted the PR :)

**Comment by polaramike** [comment]
Hi!

Maybe of interest for the Offline tools: Geotag .sub files using a GPX track.
https://github.com/polaramike/gpx2sub
